src/components/Request/Pay/Views/Initial.view.tsx

```diff
@@ -80,6 +82,7 @@ export const InitialView = ({
                 const { feeEstimation, estimatedFromAmount } = txData
                 setEstimatedFromValue(estimatedFromAmount)
                 if (Number(feeEstimation) > 0) {
+                    setErrorState({ showError: false, errorMessage: '' })
```

Comment:
Question: honestly i dont rlly get this check - why are we checking feeEstimation is > 0?

Reply:
me neither, will look into it while working on next issue